resources/rgbscripts/checkers.js

```diff
@@ -0,0 +1,360 @@
+/*
+  Q Light Controller Plus
+  checkers.js
+
+  Copyright (c) Branson Matheson
+  Based on the work of Massimo Callegari and Tim Cullingworth
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0.txt
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+// Development tool access
+var testAlgo;
+
+(
+  function()
+  {
+    var colorPalette = new Object;
+    colorPalette.collection = new Array(
+      ["White"        , 0xFFFFFF],  //  0
+      ["Cream"        , 0xFFFF7F],  //  1
+      ["Pink"         , 0xFF7F7F],  //  2
+      ["Rose"         , 0x7F3F3F],  //  3
+      ["Coral"        , 0x7F3F1F],  //  4
+      ["Dim Red"      , 0x7F0000],  //  5
+      ["Red"          , 0xFF0000],  //  6
+      ["Orange"       , 0xFF3F00],  //  7
+      ["Dim Orange"   , 0x7F1F00],  //  8
+      ["Goldenrod"    , 0x7F3F00],  //  9
+      ["Gold"         , 0xFF7F00],  // 10
+      ["Yellow"       , 0xFFFF00],  // 11
```

Comment:
Codacy has a fix for the issue: [Replace `······["Yellow"·······,·0xFFFF00],·` with `····["Yellow",·0xffff00],`](https://app.codacy.com/gh/mcallegari/qlcplus/pullRequest?prid=9305479)
```suggestion
    ["Yellow", 0xffff00], // 11
```